### PR TITLE
DB-10566 fix backward compatibility GenericColumnDescriptor

### PR DESCRIPTION
**Pull request by martinrupp**

### CODE REVIEW

File: db-engine/src/main/java/com/splicemachine/db/impl/sql/GenericColumnDescriptor.java [hunk 1]
```diff
@@ -230,9 +230,19 @@ public static String readNullableUTF(ObjectInput in) throws IOException {
      * @exception IOException thrown on error
      */
     public void writeExternal(ObjectOutput out) throws IOException {
-        writeNullableUTF(out, name);
-        writeNullableUTF(out, tableName);
-        writeNullableUTF(out, schemaName);
```
Comment (dgomezferro) [inline db-engine/src/main/java/com/splicemachine/db/impl/sql/GenericColumnDescriptor.java:235]: You can keep the `writeNullableUTF` calls for tableName and schemaName, right?

**Review by dgomezferro — APPROVED** [review]

File: db-engine/src/main/java/com/splicemachine/db/impl/sql/GenericColumnDescriptor.java (hunk 1 above)
Comment (martinrupp) [reply db-engine/src/main/java/com/splicemachine/db/impl/sql/GenericColumnDescriptor.java:235]: as it's hard to reproduce i'd rather change as little as possible

Comment (jyuanca) [reply db-engine/src/main/java/com/splicemachine/db/impl/sql/GenericColumnDescriptor.java:235]: +1 if you can test and validate

**Comment by jyuanca** [comment]
+1 if you can test and validate

**Comment by martinrupp** [comment]
> +1 if you can test and validate

i could reproduce the issue and validate that this fix fixes it.

**Review by jyuanca — APPROVED** [review]

**Review by carolp-503 — APPROVED** [review]
+1

**Review by jyuanca — APPROVED** [review]

**Pull request merged**